<launch>
  <arg name="bag_name" default="uc0102_01.bag"/>
  <arg name="scans_dir" default="/media/lch/SeagateExp/bag/sgslam/scans"/>

  <node pkg="rosbag_exchange" type="rosbag_reader_node" name="rosbag_reader_node" output="screen">

    <param name="rosbag_path" value="$(arg scans_dir)/$(arg bag_name)"/>
    <param name="rgb_topic_name" value="/camera/color/image_raw"/>
    <param name="depth_topic_name" value="/camera/aligned_depth_to_color/image_raw"/>
    <param name="camera_info_topic_name" value="/camera/aligned_depth_to_color/camera_info"/>
    <param name="camera_pose_topic_name" value="/vins_estimator/camera_pose"/>

  </node>
</launch>
